docs/writing-docs/mdx.md

```diff
@@ -211,6 +211,37 @@ Write your documentation as you usually would, and your existing SCSS code block
 
 <!-- prettier-ignore-end -->
 
+### Creating a Changelog story
+
+One common use case for MDX-only docs stories is importing a project's `CHANGELOG.md` into an MDX story, so that users can easily refer to the CHANGELOG via a documentation node in Storybook.
+
+First, ensure that `transcludeMarkdown` is set to `true` in `main.js`:
+
+```js
+addons: [
+  "@storybook/addon-links",
+  "@storybook/addon-essentials",
+  // set transcludeMarkdown to true
+  {
+    name: "@storybook/addon-docs",
+    options: { transcludeMarkdown: true },
+  },
+]
+```
+
+Then, import the markdown and treat the imported file as a component in the MDX file: 
+
+```mdx
+import { Meta } from "@storybook/addon-docs";
+
+import Changelog from "../CHANGELOG.md";
+
+<Meta title="Changelog" />
+
+<Changelog />
+```
+![Changelog markdown in an MDX story](https://user-images.githubusercontent.com/9900326/168942693-b3954bed-0369-4eb5-a4ed-634ebb7575fa.png)
+
```

Comment:
@TheeMattOliver if you wouldn't mind recapturing this image, using an example Storybook instead of the one which I assume is the one belonging to the company you work at, place it inside the current docs folder (i.e., `docs/writing-docs`) and if possible run it by [tinypng](https://tinypng.com/) to allow it to be optimized for the web, I'd appreciate it.

The reason being is that we want to maintain the documentation as agnostic as possible in terms of assets (images, videos) and go with examples relatable to the community. Adding this image as is could throw off some of our community/readers.

Reply:
Sure no worries! That was actually just some random Storybook I found on the web :) Will update!

Reply:
@TheeMattOliver thanks for addressing the changes so promptly, but looking at them it seems that you forgot to replace the image in the documentation with the one you've added with the latest commit.

This link should be:

```md
![Changelog markdown in an MDX story](./changelog-mdx-md-transcludemarkdown-optimized.png)
```

Reply:
Oops sorry about that! Done.